spec/integration/pgpapi.spec.js

```diff
@@ -3,15 +3,21 @@
 
 describe('PGP api integration', function() {
   'use strict';
-  var freedom;
+  var fdom, path;  // hack to avoid masking freedom when already defined
+  if (typeof freedom === 'undefined') {
+    fdom = require('freedom-for-node').freedom;
```

Comment:
you could just do an assignment to freedom = require('freedom-for-node').freedom
Pretty minor but that way we keep with the freedom-style semantics